doc/src/build/install.md

```diff
@@ -79,44 +81,53 @@ After you install Rust, proceed to the prerequisites for your operating system.
 
 ## Linux prerequisites 
 
-Install the prerequisites listed in this section. You should make sure that your system has the latest version of `apt`. Use the following command to update it:
-`sudo apt-get update`
+Take Ubuntu as an example, install the prerequisites listed in this section. You should make sure that your system has the latest version of `apt-get`. Use the following command to update it:
```

Comment:
`apt-get` is only for ubuntu. 
@randall-Mysten  you may want to refine the wording here, thanks!